### PR TITLE
Add xref to Web SDK in VS Publish Profiles

### LINKED ISSUE #16769: Link to Web SDK
You could add hyperlinks to the https://docs.microsoft.com/en-us/aspnet/core/razor-pages/web-sdk?view=aspnetcore-3.1 article in at least a couple places near the beginning of the article, like you do for Razor SDK.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 684d1f55-3d8b-9ba6-ee65-e8f84c2b4244
* Version Independent ID: 2db330b4-e830-715f-0a15-e406f85e1d3b
* Content: [Visual Studio publish profiles (.pubxml) for ASP.NET Core app deployment](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/visual-studio-publish-profiles?view=aspnetcore-3.1)
* Content Source: [aspnetcore/host-and-deploy/visual-studio-publish-profiles.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/visual-studio-publish-profiles.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by serpent5**
Fixes #16769.

### CODE REVIEW

File: aspnetcore/host-and-deploy/visual-studio-publish-profiles.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The `dotnet new mvc` command produces a project file containing the following ro
...
 
-`Microsoft.NET.Sdk.Web` (Web SDK) depends on other SDKs, including `Microsoft.NET.Sdk` (.NET Core SDK) and `Microsoft.NET.Sdk.Razor` ([Razor SDK](xref:razor-pages/sdk)). The MSBuild properties and targets associated with each dependent SDK are imported. Publish targets import the appropriate set of targets based on the publish method used.
+`Microsoft.NET.Sdk.Web` ([Web SDK](xref:razor-pages/web-sdk)) depends on other SDKs, including `Microsoft.NET.Sdk` (.NET Core SDK) and `Microsoft.NET.Sdk.Razor` ([Razor SDK](xref:razor-pages/sdk)). The MSBuild properties and targets associated with each dependent SDK are imported. Publish targets import the appropriate set of targets based on the publish method used.
```
Comment (serpent5) [inline aspnetcore/host-and-deploy/visual-studio-publish-profiles.md:28]: Should we add a link for the .NET Core SDK references, too? Perhaps one of these:

- https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props
- https://docs.microsoft.com/en-us/dotnet/core/sdk

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/visual-studio-publish-profiles.md:28]: Yes, why not add both links?

Comment (serpent5) [reply aspnetcore/host-and-deploy/visual-studio-publish-profiles.md:28]: I was just gonna make .NET Core SDK into a link, so I'd have to choose one the URLs.

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/visual-studio-publish-profiles.md:28]: `https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props` looks better to me, but either one is fine.

**Comment by Rick-Anderson** [comment]
@serpent5  thanks!

**Pull request merged**